tensornetwork/backends/shell/shell_backend.py

```diff
@@ -210,3 +209,16 @@ def randn(self,
 
   def conj(self, tensor: Tensor) -> Tensor:
     return tensor
+
+  def eigsh_lanczos(
+      self,
+      A: Callable,
+      initial_state: Optional[Tensor] = None,
+      ncv: Optional[int] = 200,
+      numeig: Optional[int] = 1,
+      tol: Optional[float] = 1E-8,
+      delta: Optional[float] = 1E-8,
+      ndiag: Optional[int] = 20,
+      reorthogonalize: Optional[bool] = False) -> Tuple[List, List]:
+    raise NotImplementedError(

```

Comment:
It should (?) be possible to add this to the shell backend. It's just shape calculations.

Reply:
added